Sources/Operators/CatchToResult.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CatchToResult.swift
+//  CombineExt
+//
+//  Created by Yurii Zadoianchuk on 05/03/2021.
+//  Copyright © 2021 Combine Community. All rights reserved.
+//
+
+#if canImport(Combine)
+import Combine
+
+@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+public extension Publisher where Failure: Error {
```

Comment:
I think `Failure` always needs to conform to `Error`, so `where Failure: Error` could probably be removed?